### PR TITLE
Enable CodeQL in Azure Pipelines

### PR DESCRIPTION
**Pull request by Eskibear**

### CODE REVIEW

File: .azure-pipelines/ci.yml [hunk 1]
```diff
@@ -17,7 +17,20 @@ jobs:
...
-
+  - job: CodeQL
+    displayName: Code QL
+    pool:
+      vmImage: 'windows-latest'
+    steps:
+      - task: Semmle@1
+        inputs:
+          sourceCodeDirectory: '$(Build.SourcesDirectory)'
+          language: 'tsandjs'
```
Comment (jdneo) [inline .azure-pipelines/ci.yml:28]: Will you enable it on Java?

Comment (Eskibear) [reply .azure-pipelines/ci.yml:28]: good question, let me try if I can enable java together.

Comment (Eskibear) [reply .azure-pipelines/ci.yml:29]: seems we cannot set multiple languages for the same task, so I added another task for java. hope it works.

**Comment by Eskibear** [comment]
Now I move all static analysis tasks into one job, including credscan and CodeQL for ts/js/java.

**Review by jdneo — APPROVED** [review]

**Pull request merged**